src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1599,6 +1612,11 @@ private function getCommitMessageFromUser() {
         $commit = head($this->getRepositoryAPI()->getLocalCommitInformation());
         $template = $commit['message'];
       } else {
+        $revert_plan_check_paths = $this->getConfigurationManager()->getConfigFromAnySource(
```

Comment:
@raviagarwal7 Is there some standard tool we use to indent PHP code properly?

Reply:
not that I know of, just used sublime text for it.